docs/reference/filters.md

```diff
@@ -276,6 +279,8 @@ Example:
 * -> flowId("reuse") -> "https://some-backend.example.org";
 ```
 
+[flowid_validation]: https://github.com/zalando/skipper/blob/b2d74584a5753b21034be452743ae242f86fd513/filters/flowid/ulid.go#L55-L58
```

Comment:
from the code pov, there are 3 conditions, but only the regex is important.
Regexp says `^[0123456789ABCDEFGHJKMNPQRSTVWXYZ]{26}$`, which means `^[0-9A-Z]{26}$`, so numbers and upper ascii alphabet with exact 26 characters. Maybe the len check is done to make sure we have no slow regex, but then we could do `len(fid) == 26 && regexCheck`

Reply:
Changed PTAL

Reply:
I was even wrong in which generator is used by default. Now it should be more clear and correct in the docs, thanks!

Reply:
Could not simplify the regex to `^[0-9A-Z]{26}$ ` because the original doesn't have a `U` in the list of characters and tests are failing with that change. I reverted that part.

Reply:
ah oh :)